projects/plugins/jetpack/modules/contact-form.php

```diff
@@ -18,6 +21,11 @@
  * Additional Search Queries: contact, form, grunion, feedback, submission, contact form, email, feedback, contact form plugin, custom form, custom form plugin, form builder, forms, form maker, survey, contact by jetpack, contact us, forms free
  */
 
+if ( defined( 'JETPACK_BETA_BLOCKS' ) && JETPACK_BETA_BLOCKS ) {
```

Comment:
In my experience this doesn't work on .com, just leaving a note so we don't forget.

See:
- p1669932430568779-slack-CDLH4C1UZ

Reply:
This should work on WordPress.com Simple (the flag is set when you're a proxied Automattician), but I would recommend using a specific filter to gate access, one that is only used for forms. This is what Paul mentioned in the thread.